Gemfile

```diff
@@ -5,9 +5,9 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-gem 'conjur-api', '~> 5.1.0'
-gem 'activesupport', '~> 5.2.0'
-gem 'railties', '~> 5.2.0'
+gem 'conjur-api', '~> 5.3.1'
+gem 'activesupport', '~> 5.2.1'
+gem 'railties', '~> 5.2.1'
```

Comment:
These last two lines don't actually affect the gem version that ends up in `Gemfile.lock`. You could have left them as-is and bundler still would have grabbed `5.2.1` since saying `~> 5.2.0` is equivalent to `>= 5.2.0` and `< 5.3`. Whatever though, doesn't hurt anything either! :shipit:

Reply:
fair enough :) I forget the syntax, but I would've gotten that if it just said `5.2` (eg I would have then assumed it was 5.2.x)